cookbook/docs/deployment.rst

```diff
@@ -6,6 +6,6 @@ Deployment
     :maxdepth: -1
     :caption: Deployment
  
-    Workflow Deployment <auto/deployment/workflow/index>
+    workflow_deployment
```

Comment:
I think we should use the `CUSTOM_FILE_SORTER` to sort the order of the autogenerated pages:
https://github.com/flyteorg/flytesnacks/blob/master/cookbook/docs/conf.py#L67-L72

It's easier to maintain that way because we don't have to manually create/update new `.rst` files just to sort pages